### ISSUE TITLE
Picture orientation issue

### ISSUE THREAD

**Issue by karan-saluja** [engram-design/ImageResizer#60]
The Plugin sometimes rotates a portrait image into a landscape one. 
My Image driver: "Imagick 3.4.4"
Please try using this image:
![2020-10-27 13 34 01](https://user-images.githubusercontent.com/28668444/118123245-99a36c80-b411-11eb-86f9-41497739b719.jpg)

**Comment by dgsiegel** [comment]
This seems to be a problem with image-resizer and removed EXIF orientation tags. Currently I've set up the plugin to resize everything to 2400x2400 pixels. My craft config features these two lines:

```
'preserveExifData' => false,
'rotateImagesOnUploadByExifData' => true,
```

Now the funny thing is that all oriented images below 2400px are displayed correctly (i.e. EXIF removed and rotated correctly), but all above 2400px are not. The only solution I've found would be to set `'preserveExifData' => true`.

If you add the following lines to https://github.com/verbb/image-resizer/blob/craft-3/src/services/Service.php#L133:

```
$i = new \Imagick($filePath);
$orientation = $i->getImageOrientation();
ImageResizer::$plugin->logs->resizeLog(null, 'error', $filePath, ['message' => 'orientation ' . print_r($orientation, true)]);
```

you can see that `$orientation` will be empty, except when setting `'preserveExifData' => true`. 

So it seems that craft will remove EXIF tags and hence they cannot be used by image resizer anymore.

**Comment by engram-design** [comment]
So that sort of makes sense, because I believe `$image->saveAs()` will strip Exif data. But the function `rotateImageByExifData()` won't kick in until after we've resized the image. But because the Exif data has already been stripped out, it can't be rotated properly. We need to call `saveAs()` before Craft does, because the plugin is resizing the image, but as I've explained, that'll mess up Exif data.

This _used_ to work sometime ago, but that's been largely unchanged since Craft 3.

Should be fixed in [2.1.1](https://github.com/verbb/image-resizer/releases/tag/2.1.1)
